src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/endpoints/IdamConfigStatusEndpointTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.endpoints;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.config.IntegrationTest;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.model.out.JurisdictionConfigurationStatus;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+
+@IntegrationTest
+@AutoConfigureMockMvc
+public class IdamConfigStatusEndpointTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @DisplayName("Should have BULKSCAN dummy jurisdiction configured properly")
+    @Test
+    public void should_have_bulkscan_configured_properly() throws Exception {
```

Comment:
should it be not for all configured juridictions ?

Reply:
it is integration test. so whatever is configured as per dev environment aka `integration` [profile (see the yaml here)](https://github.com/hmcts/bulk-scan-orchestrator/tree/master/src/test/resources)

everything is checked in smoke test